tests/projects/test_project_spec.py

```diff
@@ -13,15 +13,18 @@ def test_project_get_entry_point():
     project = load_project()
     entry_point = project.get_entry_point("greeter")
     assert entry_point.name == "greeter"
-    assert entry_point.command == "python greeter.py {greeting} {name}"
+    assert entry_point.command == "python greeter.py {greeting} {name} {args}"
     # Validate parameters
-    assert set(entry_point.parameters.keys()) == set(["name", "greeting"])
+    assert set(entry_point.parameters.keys()) == set(["name", "greeting", "args"])
     name_param = entry_point.parameters["name"]
     assert name_param.type == "string"
     assert name_param.default is None
     greeting_param = entry_point.parameters["greeting"]
     assert greeting_param.type == "string"
     assert greeting_param.default == "hi"
+    greeting_param = entry_point.parameters["args"]
```

Comment:
```suggestion
    action_param = entry_point.parameters["args"]
```

Nit: recommend making this separate from the variable name above.